### PR TITLE
Cleanup of rdkit.Chem.Fingerprints

### PR DESCRIPTION
**Pull request by gedeck**
I try to make the changes to code easier to review by splitting it up into several PRs. This PR covers minor formatting and standardisation changes.

### CODE REVIEW

File: rdkit/Chem/Fingerprints/ClusterMols.py [hunk 1]
```diff
@@ -18,16 +17,21 @@
...
+
+from rdkit import DataStructs
+from rdkit.Chem.Fingerprints import FingerprintMols, MolSimilarity
 from rdkit.ML.Cluster import Murtagh
-import sys
 from rdkit.six.moves import cPickle
-from rdkit.Chem.Fingerprints import FingerprintMols, MolSimilarity
-from rdkit import DataStructs
-import numpy
+
```
Comment (greglandrum) [inline rdkit/Chem/Fingerprints/ClusterMols.py]: If this block of imports isn't needed, why not just remove them?

File: rdkit/Chem/Fingerprints/ClusterMols.py [hunk 2]
```diff
@@ -18,16 +17,21 @@
...
+# from rdkit.Dbase.DbConnection import DbConnect
+# from rdkit.Dbase import DbInfo, DbUtils
+# from rdkit.ML.Data import DataUtils
+# from rdkit.ML.Cluster import Clusters
+# import sys
+
 _cvsVersion = "$Id$"
```
Comment (greglandrum) [inline rdkit/Chem/Fingerprints/ClusterMols.py]: Wow, this is a horrible ancient source control dropping. 
I think we can get rid of everything between here and the definition of `message`. The usage of __VERSION_STRING way below in the "main" block can also be removed.

File: rdkit/Chem/Fingerprints/MolSimilarity.py [hunk 3]
```diff
@@ -16,25 +15,24 @@
...
-from rdkit.Chem.Fingerprints import FingerprintMols, DbFpSupplier
+
 try:
   from rdkit.VLib.NodeLib.DbPickleSupplier import _lazyDataSeq as _dataSeq
 except ImportError:
   _dataSeq = None
 
-from rdkit import DataStructs
-
 _cvsVersion = "$Id$"
```
Comment (greglandrum) [inline rdkit/Chem/Fingerprints/MolSimilarity.py]: Same comment here about removing this cruft.

**Comment by gedeck** [comment]
CVS information and commented import statements removed.

**Pull request merged**